x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result_field_value.tsx

```diff
@@ -22,10 +22,7 @@ const getRawArrayDisplay = (rawArray: Array<string | number>): string => {
 };
 
 const parseHighlights = (highlight: string): string => {
-  return highlight.replace(
-    /<em>(.+?)<\/em>/gi,
-    '<em class="enterpriseSearchResultHighlight">$1</em>'
-  );
+  return highlight.replace(/<em>(.+?)<\/em>/gi, '<mark class="euiMark">$1</mark>');
```

Comment:
This was a suggestion from a [previous PR](https://github.com/elastic/kibana/pull/83564#discussion_r527918117)

Reply:
This was actually super interesting because I was expecting my screenreader (VO) to in some way announce or emphasize the `<mark>`'d content but it looks like SR support for mark is actually not super great:

https://developer.paciellogroup.com/blog/2017/12/short-note-on-making-your-mark-more-accessible/

Fun times in screen reader land 🤦‍♀️ I'm still a strong +1 for this as I think it's a great change and definitely wanna make use of EuiMark.